### PR TITLE
Revert of AI service changes

### PR DESCRIPTION
**Pull request by BarryJRowe**
Potential fix for issue #16275 by reverted to previous version of the AI service.

@bslenul Please test to see if it works for you. It seems to work on my system for HW, but in my case it was just returning a black screen for HW cores, not upside down images. I also have a limited number of video drivers available to me to test against.

### CODE REVIEW

**Comment by bslenul** [comment]
Works fine AFAICT on Windows10, tested with Flycast, Beetle PSX HW, Mupen64Plus-Next, PPSSPP (crashed once but knowing how stable Vulkan is with that core I'm not surprised), LRPS2, Dolphin and Citra.

Compilation is failing however because of a missing `;` in `return false` in platform_win32.c, line 1087.

A shame that it has to be reverted, but it's better than its current broken behavior.

File: frontend/drivers/platform_win32.c [hunk 1]
```diff
@@ -1081,12 +1082,9 @@ static bool accessibility_speak_windows(int speed,
       if (!wc || res != 0) 
       {
          RARCH_ERR("Error communicating with NVDA\n");
-         /* Fallback on powershell immediately and retry */
-         g_plat_win32_flags &= ~PLAT_WIN32_FLAG_USE_NVDA;
-         g_plat_win32_flags |= PLAT_WIN32_FLAG_USE_POWERSHELL;
          if (wc)
             free(wc);
-         return accessibility_speak_windows(speed, speak_text, priority, voice);
+         return false
```
Comment (bslenul) [inline frontend/drivers/platform_win32.c:1087]: Missing `;`

**Comment by viachaslavic** [comment]
Known issues on GNU/Linux:
* `Text Mode` and `Text + Narrator` modes dropped out of the UI
* Worked only `Image mode`:
   * `Speech Mode` returns **[ERROR] Download failed: Invalid JSON body.**
   * `Narrator Mode` does nothing and don't write anything in the log

**Comment by BarryJRowe** [comment]
> Known issues on GNU/Linux (possibly on other platforms as well):
> 
> * `Text Mode` and `Text + Narrator` modes dropped out of the UI
> * Worked only `Image mode`:
>   
>   * `Speech Mode` returns **[ERROR] Download failed: Invalid JSON body.**
>   * `Narrator Mode` does nothing and don't write anything in the log

The breaking PR added those new modes (text and text+narrator), so they were removed as options. Narrator looks to be working on my linux machine, but needs accessibility enabled (narrator uses espeak on linux, and nvda on windows).

Speech mode is confirmed not working, so I'll have a look into that today.

**Comment by zoltanvb** [comment]
Checked on Linux + wayland + glcore + Beetle PSX HW, works.
I do hope new modes can be re-introduced in due time. Can the changes under intl/ and in msg_hash.h be excluded? Those should not harm anything, and it also cuts down on the change amount.

**Comment by BarryJRowe** [comment]
@viachaslavic I've confirmed that speech mode is actually working (all gae audio on my system was not working) once I switched to another computer, it was working fie. If you're still having an issue with speech mode, then please post the details of how you're getting that error (example screenshot, core, etc.)

**Comment by viachaslavic** [comment]
@BarryJRowe you're right, `Speech Mode` behave the same as current master. Issue is related to unsupported languages as `Target Language`. In this case the service simply returns an empty json. Moreover, when reverting, an error **[ERROR] Download failed: Invalid JSON body** is written to the log.  Current master doesn't react to this at all.

**Pull request merged**